network/transport/grpc/connection.go

```diff
@@ -55,7 +63,9 @@ type Connection interface {
 
 	// Send tries to send the given message over the stream of the given protocol.
 	// If there's no active stream for the protocol, or something else goes wrong, an error is returned.
-	Send(protocol Protocol, envelope interface{}) error
+	// A sender may specify overdrive=true to allow extra messages to be sent.
+	// This is needed to finish sending a TransactionList that falls within a single page.
+	Send(protocol Protocol, envelope interface{}, overdrive bool) error
```

Comment:
overdrive? 🚀 

godoc

I had to dig into the rest of the changes to see when it's used, I think overdrive is a (funny, but) bad term. Maybe just `ignoreOutboxLimit`? Actually, the kind of messages that may wish to ignore the soft limit are  "burst-like" messages/batch (transaction list).